syncer/syncer.go

```diff
@@ -836,7 +824,7 @@ func (s *Syncer) addJob(job *job) error {
 	case skip:
 		s.updateReplicationLag(job, skipLagKey)
 	case flush:
-		addedJobsTotal.WithLabelValues("flush", s.cfg.Name, adminQueueName, s.cfg.SourceID).Inc()
+		s.addCount(false, adminQueueName, job.tp, 1, job.targetSchema, job.targetTable)
```

Comment:
seems we could only left one line

Reply:
i can not understoond this mean, L827's addCount use `addedJobsTotal` but L841's addCount use `finishedJobsTotal`